eng/Signing.props

```diff
@@ -5,5 +5,31 @@
     <AllowEmptySignList>true</AllowEmptySignList>
     <AllowEmptySignPostBuildList>true</AllowEmptySignPostBuildList>
   </PropertyGroup>
+  
+  <ItemGroup>
+    <!--
+      Replace the default items to sign with the specific set we want. This allows the build to call
+      Arcade's Sign.proj multiple times for different sets of files as the build progresses.
+    -->
+    <ItemsToSign Remove="@(ItemsToSign)" />
+
+    <!-- Exclude symbol packages from getting a NuGet signature. These are never pushed to NuGet.org or
+         other feeds (in fact, that have identical identity to their non-symbol variant) -->
```

Comment:
Why do we care about that? Other repositories don't have this and just use the Arcade default behavior.

Reply:
I copied that comment from runtime. Sounds like we want to avoid wasting resources. Don't know if we have any other reasons to do that in _runtime_ but it sounded like an acceptable reason to copy that here too.

Reply:
I don't think we should do any of that. If this is useful then that logic should be pushed into Arcade but maintenance-packages shouldn't need any signing specialization here.